.php_cs

```diff
@@ -0,0 +1,33 @@
+<?php
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__ . "/src");
+
+return PhpCsFixer\Config::create()
+    ->setRules(
+        [
+            '@PSR2'                               => true,
+            'array_syntax'                        => [
+                'syntax' => 'short'
+            ],
+            'class_keyword_remove'                => false,
+            'concat_space'                        => [
+                'spacing' => 'one'
+            ],
+            'combine_consecutive_unsets'          => true,
+            'general_phpdoc_annotation_remove'    => [
+                'annotations' => ['@author'],
+            ],
+            'linebreak_after_opening_tag'         => true,
+            'no_short_echo_tag'                   => true,
+            'no_useless_else'                     => true,
```

Comment:
@qurben @andrederoos, @dnl-blkv if you prefer to keep the else statements, I will remove the commit. But in my opinion, a difference of 1700 lines of code is also readability. If 'this' situation occurs, then we will do 'this'. In all other cases, the default flow will be handled. So, when you make an opinion, I will apply it and close the further discussion.